logger/logger_types.go

```diff
@@ -28,6 +30,9 @@ var (
 
 	// LogPath system path to store log files in
 	LogPath string
+
+	f = func(f bool) *bool { return &f }(false)
+	t = func(t bool) *bool { return &t }(true)
```

Comment:
This stuff is becoming a bit more common. Would you be able to consolidate these?

Reply:
I did move this into the common (and also utils) package at one point but ran into a bunch of import cycles i didn't want to create just a single package for it at the moment but will see if there is anything suitable that doesn't cause import issues

Reply:
a commit is coming that creates a new function called BoolPtr in common/convert that takes the condition and returns a pointer to it that replaces usage of these